swift/WorkflowTesting/Sources/WorkflowRenderTester.swift

```diff
@@ -230,7 +232,7 @@
     fileprivate final class RenderTestContext<T: Workflow>: RenderContextType {
         typealias WorkflowType = T
 
-        private var (lifetime, token) = Lifetime.make()
+        private var (lifetime, token) = ReactiveSwift.Lifetime.make()
```

Comment:
Do we dislike the need to disambiguate enough to use a different name than `Lifetime`? I couldn’t come up with anything better when I originally typed my version of this, but wanted to raise the question in case others had thoughts.

Reply:
I don't think we need to rename just to disambiguate. As we continue to remove the `ReactiveSwift` dependency from the core library - this will become less of an issue.